Documentation/ceph-common-issues.md

```diff
@@ -768,3 +768,15 @@ This problem will be solve by the following two fixes.
 * Ceph: A fix that uses the above-mentioned kernel's feature. The Ceph community will probably discuss this fix after releasing Linux v5.6.
 
 You can bypass this problem by using ext4 or any other filesystems rather than XFS. Filesystem type can be specified with `csi.storage.k8s.io/fstype` in StorageClass resource.
+
+## Too few PGs per OSD warning is shown
+
+### Symptoms
+
+- `ceph status` shows "too few PGs per OSD" warning.
```

Comment:
How about showing a snippet of the output from `ceph status` in a multi-line comment so it's clear that this is a ceph health warning?

Reply:
Added the result of `ceph status` as you said.